### PR TITLE
Add post for favorites list

### LINKED ISSUE #11: Create POST request in server.js

### PR DESCRIPTION
**Pull request by Fordo29**
## What Features are Being Added?
Add Post for adding a book to the favorites data set
Add error handling to post when an incomplete request is made

## What is Being Refactored?
nothing at this time

## Where should the reviewer start?
server.js

## How should this be manually tested?
Postman

## What are the relevant tickets?
Relevant ticket is creating a post request and error handling

## Any Other Notes/Questions/Problems?
Not at this time

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -30,6 +35,20 @@ app.get('/api/v1/books/:id', (request, response) => {
   response.status(200).json(book);
 });
 
+app.post('/api/v1/favorites', (request, response) => {
```
Comment (krdurrett) [inline server.js:38]: Nice formatting for destructuring and error handling.

**Review by krdurrett — APPROVED** [review]
This is a great post request

File: server.js (hunk 1 above)
Comment (cagallo) [reply server.js:38]: Fantastic error handling!

**Review by cagallo — COMMENTED** [review]
This POST request is fantastic Christine!

**Comment by ectweitmann** [comment]
Wow look at the speed with which we are writing this code. We're killing it! Go team!

**Pull request merged**